### ISSUE TITLE
Normals arent imported

### ISSUE THREAD

**Issue by aronwk-aaron** [sttng/ImportLDD#1]
I'm looking to have normals imported for renders to look better in multi-geo parts
I see there is some code in there for normals, but after following the codepath, it looks like it is legacy and never used.
Here is an example of the issue with part 30124
![Screenshot from 2022-03-03 15-16-41](https://user-images.githubusercontent.com/26027722/156654366-3902c477-91ae-4345-be56-0ebd93c07c88.png)

**Comment by sttng** [comment]
Yeah, I think you are right. I hacked this whole thing together only in a couple of hours overall.

**Comment by aronwk-aaron** [comment]
All good! I'm confident with python, but not that familiar with blender.

building the normals list like:
```python
                        normals = []
                        for point in geo.Parts[part].outnormals:
                            single_norm = mathutils.Vector([point.x, point.y, point.z])
                            normals.append(single_norm)
```
I tried importing the normals like this:
https://docs.blender.org/api/current/bpy.types.Mesh.html#bpy.types.Mesh.normals_split_custom_set_from_vertices
but with no luck.

But then tried this, which is an older way to do it (i've lost the stackoverflow where I found this)
```python
                        vindex = 0
                        for vertex in mesh.vertices:
                            vertex.normal = normals[vindex]
                            vindex += 1
```
which works, but the normals get nuked whenever you go into edit mode.

**Comment by sttng** [comment]
I remember that I found the normals from LDDs geometry files (.g, .g1, .g2, ...) weird once in a while. Thats why I added a no-normals setting flag for my LDD-> Renderman (LegoToR) exporter. Also honestly I assumed the normals to be face normals, but maybe they are something else.

Finally, similar to you I never found a good way to import normals into Blender and I found the documentation lacking :( I think I even looked into the obj importer of blender but couldn't grasp a good way. Maybe I'll take a look again :)
